code-challenges.js

```diff
@@ -1,63 +1,117 @@
-
+//
+// // put a const so I can console log it later
+const divisibleOrnot = (num) => {
+// // Create an equation so whenever I have a no remainder it'll spit out is divisible by 3
+    if(num % 3 == 0){
+        return `${num} Divisible by 3`
+    }else{
+        return `${num} Not divisible by 3`
+    }
+}
+// // these syntaxes are gonna be the end of me
+console.log(divisibleOrnot(num1));
+console.log(divisibleOrnot(num2));
+console.log(divisibleOrnot(num3));
+//
+//
+// // Expected output: "-7 is not divisble by three"
+// // Expected output: "15 is divisible by three"
+// // Expected output: "0 is divisible by three"
+//
+//
+//
+// // --------------------2) Create a function that takes in the randomNouns variable and returns an array with all the words capitalized.
+//
 var randomNouns = ["streetlamp", "potato", "teeth", "conclusion", "nephew", "temperature", "database"]
-// Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew", "Temperature", "Database"]
-
-
-
-
+//
+// // I'm declaring a function that takes in this array ^ and returns the first letter capitalized.
+  const capitalizer = (array) => {
+// // So I have to declare a function that takes in the array
+    let eachItem = array.map(value => {
+// // Then access the first index and capalize the letter
+    return value[0].toUpperCase() + value.substring(1)
+  })
+// // return the local variable with a .join so itll go back into a string
+  return eachItem.join(" ")
+}
+console.log(capitalizer(randomNouns));
+//
+// // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew", "Temperature", "Database"]
+//
+// // --------------------3) Create a function that takes in the mixedDataArray array and returns an array with ONLY NUMBERS sorted from least to greatest.
+//
+var mixedDataArray = [true, 8, "hello", 90, -8, null, 0, 46, 59, 107, "hey!"]
+//
+const onlyNums = (array)=>{
+  let filteredArray = array.filter(value=> {
+    return (typeof value === "number")
+  })
+      return filteredArray.sort(function(a, b){return a-b})
+}
+console.log(onlyNums(mixedDataArray));
+//
+//
+//
+//
+// // Expected output: [-8, 0, 8, 46, 59, 90, 107]
 
-// --------------------3) Create a function that takes in the mixedDataArray array and returns an array with ONLY NUMBERS sorted from least to greatest.
 
-var mixedDataArray = [true, 8, "hello", 90, -8, null, 0, 46, 59, 107, "hey!"]
-// Expected output: [-8, 0, 8, 46, 59, 90, 107]
 
+// --------------------4) Create a function that takes in a string and logs the index of the first vowel.
+// At first we would want to create a funtion thatll take in voweltester1 and 2 and split them.
+// use a fitler to get rid of the vouls a,e,i,o,u,y and then do a .join at the end
 
+//
+//   const str = ("";
+//   var res = str.split(`a` && `e` && `i` && `o` && `u`);
+// )
 
+// I'm not 100% sure if the sytaxes are right but I wanna say I'm close so then after that I would .join it afterwords
 
+// console.log(str(res));
```

Comment:
Not quite. Again, we want to make sure this is a function based upon the challenge description as of right now, this code does not work and is not acceptable to the challenge requirements.  Please be sure to read these thoroughly.